### PR TITLE
Import cleanup

### PR DESCRIPTION
**Pull request by djnunez-aot**
*Issue #225:*
https://github.com/bcgov/met-public/issues/#225

Cleanup import to make code more readable

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Review by tiagograf-AOT — APPROVED** [review]

**Review by saravanpa-aot — APPROVED** [review]
Good stuff..Please wait for review from @jadmsaadaot as well

**Pull request merged**
